res/skins/QMLDemo/main.qml

```diff
@@ -51,6 +51,62 @@ Rectangle {
 
         }
 
+        WaveformRow {
```

Comment:
For Qt 5.15 this entire section could be further shortened by using an inline component as a template.

Reply:
Turns out, you don't. I just found the relevant docs for it.